### PR TITLE
Reset pbem token - Fixes bug #1464

### PR DESCRIPTION
**Pull request by pseudomonas**
This should fix bug 1464.

### CODE REVIEW

**Comment by kareila** [comment]
Hey @pseudomonas, I wanted to let you know that we ask our contributors to follow the syntax outlined in https://help.github.com/articles/closing-issues-via-commit-messages/ which is picky about what it accepts. "This should fix #1464" works for making sure the issue gets assigned to you and closed when the PR is merged, but the similar syntax you used above doesn't. Sorry for the trouble :)

File: cgi-bin/DW/Console/Command/ResetPBEMToken.pm [hunk 1]
```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/perl
+#
+# DW::Console::Command::ResetPBEMToken
+#
+# Console command for resetting PBEM tokens
+#
+# Authors:
+#      Adam Bernard <user name="pseudomonas" site="dreamwidth.org"">
```
Comment (kareila) [inline cgi-bin/DW/Console/Command/ResetPBEMToken.pm]: Don't use the user tag syntax here. The URL syntax you used in your previous PR is fine, though!

File: cgi-bin/DW/Console/Command/ResetPBEMToken.pm [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    return $self->error("This command takes two arguments. Consult the reference.")
+        unless $username && $reason && scalar(@args) == 0;
+
+    my $u = LJ::load_user($username);
+    return $self->error("Invalid user $username")
+        unless $u;
+
+    my $newauth = $u->generate_emailpost_auth();
+
+    return $self->print( "Token not reset" ) unless $newauth;
```
Comment (kareila) [inline cgi-bin/DW/Console/Command/ResetPBEMToken.pm]: Should this be `return $self->error` instead of `return $self->print`?

**Comment by kareila** [comment]
This seems to work fine! If you'll address the two minor issues I brought up, I'll get this merged. Thanks!

File: cgi-bin/DW/Console/Command/ResetPBEMToken.pm (hunk 2 above)
Comment (rahaeli) [inline cgi-bin/DW/Console/Command/ResetPBEMToken.pm]: Yes, please use $self->error, it prints differently and someone (*raises hand*) is a Bear of Very Little Brain and does not always notice that something didn't succeed unless it's printed in error styling. :)

**Comment by kareila** [comment]
Sorry, I just saw you updated this! Adding a commit to an open PR doesn't fire off a notification, sadly!

**Pull request merged**